src/screens/MessageScreen.tsx

```diff
@@ -150,6 +179,16 @@ const defaultStyles = createStyles('MessageScreen', (theme) => {
     },
     badgeView: {
       alignSelf: 'center',
+      paddingRight: 0,
+      marginRight: -10,
+    },
+    badgeColor: {
+      enabled: {
+        backgroundColor: theme.colors.primary,
```

Comment:
A consumer of this can set this color to transparent if they don't want the badge.

Reply:
Or they can do `badgeView.display: 'none'` That's my go to for hiding content.